docs/04-operation-guides/troubleshooting/README.md

```diff
@@ -16,6 +16,47 @@ See the full list of Kyma troubleshooting guides:
+- [Issues with Istio Sidecar Injection](https://kyma-project.io/#/istio/user/troubleshooting/03-30-istio-no-sidecar)
+- [Incompatible Istio Sidecar Version After Istio Operator’s Upgrade](https://kyma-project.io/#/istio/user/troubleshooting/03-40-incompatible-istio-sidecar-version)
+- [Istio Unintentionally Removed](https://kyma-project.io/#/istio/user/troubleshooting/03-50-recovering-from-unintentional-istio-removal)
+- [Kyma Endpoint Returns a not found Error (404 Status Code)](https://kyma-project.io/#/istio/user/troubleshooting/03-60-404-on-istio-gateway)
+
+## Serverless Module
+
+- [Failure to Build Functions](https://kyma-project.io/#/serverless-manager/user/troubleshooting-guides/03-10-cannot-build-functions)
+- [Failing Function Container](https://kyma-project.io/#/serverless-manager/user/troubleshooting-guides/03-20-failing-function-container)
+- [Functions Failing to Build on k3d](https://kyma-project.io/#/serverless-manager/user/troubleshooting-guides/03-40-function-build-failing-k3d)
+- [Serverless Periodically Restarting](https://kyma-project.io/#/serverless-manager/user/troubleshooting-guides/03-50-serverless-periodically-restaring)
+
+## Telemetry Module
+
+- [Traces Troubleshooting](https://kyma-project.io/#/telemetry-manager/user/03-traces?id=troubleshooting)
+- [Metrics Troubleshooting](https://kyma-project.io/#/telemetry-manager/user/04-metrics?id=troubleshooting)
+
+## Eventing Module
+
+- [Kyma Eventing - Basic Diagnostics](https://kyma-project.io/#/eventing-manager/user/troubleshooting/evnt-01-eventing-troubleshooting)
+- [NATS JetStream Backend Troubleshooting](https://kyma-project.io/#/eventing-manager/user/troubleshooting/evnt-02-jetstream-troubleshooting)
+- [Subscriber Receives Irrelevant Events](https://kyma-project.io/#/eventing-manager/user/troubleshooting/evnt-03-type-collision)
+- [Eventing Backend Stopped Receiving Events Due To Full Storage](https://kyma-project.io/#/eventing-manager/user/troubleshooting/evnt-04-free-jetstream-storage)
+- [Published Events Are Pending in the Stream](https://kyma-project.io/#/eventing-manager/user/troubleshooting/evnt-05-fix-pending-messages)
+
+## API Gateway Module
+
+- [Certificate Management - Issuer Not Created](https://kyma-project.io/#/api-gateway/user/troubleshooting-guides/03-20-cert-mgt-issuer-not-created)
+- [Kyma Gateway - Not Reachable](https://kyma-project.io/#/api-gateway/user/troubleshooting-guides/03-30-gateway-not-reachable)
+- [Issues When Creating an APIRule - Various Reasons](https://kyma-project.io/#/api-gateway/user/troubleshooting-guides/03-40-api-rule-troubleshooting)
+- [Issues with Certificates on Gardener](https://kyma-project.io/#/api-gateway/user/troubleshooting-guides/03-50-certificates-gardener)
+- [Cannot Connect to a Service Exposed by an APIRule - Basic Diagnostics](https://kyma-project.io/#/api-gateway/user/troubleshooting-guides/03-00-cannot-connect-to-service/03-00-basic-diagnostics)
```

Comment:
```suggestion
- [Cannot Connect to a Service Exposed by an APIRule - Basic Diagnostics](https://kyma-project.io/#/api-gateway/user/troubleshooting-guides/03-00-cannot-connect-to-service/03-00-basic-diagnostics)
  - [401 Unauthorized or 403 Forbidden](https://kyma-project.io/#/api-gateway/user/troubleshooting-guides/03-00-cannot-connect-to-service/03-01-401-unauthorized-403-forbidden?id=_401-unauthorized-or-403-forbidden)
  - [404 Not Found](https://kyma-project.io/#/api-gateway/user/troubleshooting-guides/03-00-cannot-connect-to-service/03-02-404-not-found?id=_404-not-found)
  - [500 Internal Server Error](https://kyma-project.io/#/api-gateway/user/troubleshooting-guides/03-00-cannot-connect-to-service/03-03-500-server-error?id=_500-internal-server-error)
```
This folder contains a few subdocuments. I'm not sure if we want to include them